RELEASE.md

```diff
@@ -1,3 +1,24 @@
+# Release 0.12.0
+
+## Major Features
+* Add Windows support for tensorflow-io.
+* Add PostgreSQL server dataset support.
+* Add MP3 format for audio dataset support.
+* Add ArrowIOTensor with input from a pyarrow.Table.
+* Add Numpy dataset support for numpy file and memory input.
+* Add decode_avro/encode_avro for Avro serialization support.
+* Prometheus dataset now outputs structued shapes.
+
+## Thanks to our Contributors
+
+This release contains contributions from many people:
+
+Bryan Cutler, DylanTallchiefGit, Kota Yamaguchi, Mark Daoust,
```

Comment:
Use `@DylanTallchiefGit` for users without a name?

Reply:
@terrytangyuan We respect contributor's preference on GitHub handle/name. We can always update later if contributor decide to make a change in handle/name.

Reply:
Sounds good